packages_rs/nextclade/src/run/nextclade_run_one.rs

```diff
@@ -149,12 +149,17 @@ pub fn nextclade_run_one(
 
   let aa_changes_groups = group_adjacent_aa_subs_and_dels(&aa_substitutions, &aa_deletions);
 
+  let total_aligned_nucs = alignment_end - alignment_start;
+  let total_covered_nucs = total_aligned_nucs - total_missing - total_non_acgtns - total_deletions + total_insertions;
+  let coverage = total_covered_nucs as f64 / ref_seq.len() as f64;
+
```

Comment:
The formula needs scientific review and may need adjustments

Reply:
I think it's better to take insertions out - deletions are arguable. I'm not sure what bioinfo veterans would expect. Maybe @tseemann can help here?

Since one can simply delete insertions oneself we may want to go with this instead
```rust
let total_covered_nucs = total_aligned_nucs - total_missing - total_non_acgtns;
```

Reply:
> alignment_end and alignment_start are positions on the reference, right?

Yes

> whether total_missing and total_non_acgtns count bases that are in insertions or not

No. Insertions historically are living their own life. They are stripped in Nextalign (called in the beginnign of this function) and we never touch them.

Might be something to discuss with Richard.